app/models/agents/user_location_agent.rb

```diff
@@ -6,19 +6,17 @@ class UserLocationAgent < Agent
 
     gem_dependency_check { defined?(Haversine) }
 
-    description do
-      <<-MD
-        #{'## Include `haversine` in your Gemfile to use this Agent!' if dependencies_missing?}
-        The UserLocationAgent creates events based on WebHook POSTS that contain a `latitude` and `longitude`.  You can use the [POSTLocation](https://github.com/cantino/post_location) or [PostGPS](https://github.com/chriseidhof/PostGPS) iOS app to post your location.
+    description <<-MD
```

Comment:
My fault for being an idiot there - variable scope works differently in heredocs. I'm going to see if I can find a way around that, just for consistency's sake.